jerry-core/profiles/README.md

```diff
@@ -0,0 +1,38 @@
+
+### About profile files
+
+Specify compile definitions in profile files to use when compiling the `jerry-core` target.
+
+The default profile is ``es5.1`` which disables the ArrayBuffer built-in.
+
+### Using profiles with the build system
+
+You can add profile for the build system in two way:
+  * with absolute path
+  * with a name (in this case the given profile name is assumed in the `profiles` folder)
+
+#### Restrictions
+
+The profile files can contain comments, but cannot have multi-line compile definitions, and any comma characters.
```

Comment:
Only single line options are allowed in the profile file. Any line starting with hash-mark is ignored. Semicolon character is not allowed.